source/WikiSource/Home.md

```diff
@@ -1,7 +1,11 @@
-Welcome to the documentation wiki for the Sampler DSC Pipeline
+# Welcome to the Sampler DSC Pipeline wiki
 
-# Usage
+<sup>*Sampler.DscPipeline v0.2.0-preview0013*</sup>
```

Comment:
I don't think that's how it works.
I believe it's looking for vX.X.X and then replace.
We'll see.